low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,42 @@ import sys
 import socket
 import json
 import subprocess
+import warnings
+from framework.utils.conf_utils import (GLOBAL_CONF, Conf,
+    STORAGE_TYPE_KEY, SITE_ID_KEY, RACK_ID_KEY,
+    NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.base.sspl_constants import DATA_PATH
 
-from cortx.utils.conf_store import Conf
-from framework.utils.conf_utils import Utility
-from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                        RESOURCE_HEALTH_VIEW,
-                                        sspl_config_path,
-                                        GLOBAL_CONFIG_INDEX)
+warnings.filterwarnings(action='ignore', module='cryptography')
```

Comment:
cant suppress warnings, address the issues faced here

Reply:
Removed the code